prometheus_hardware_exporter/config.py

```diff
@@ -37,6 +39,28 @@ def validate_level_choice(cls, level: str) -> str:  # noqa: N805 pylint: disable
             raise ValueError(msg)
         return level
 
+    @validator("disable_collector")
+    def validate_disable_collector_choice(  # pylint: disable=E0213
+        cls, disable_collector: List[str]  # noqa: N805
```

Comment:
Why using cls here? This is not class method or it is?

Reply:
Please refer to [pydantic's documentation](https://docs.pydantic.dev/latest/usage/validators/)

Reply:
I see, but still I will rather take a look how we can fix it, solution for flake8 is in [#568](https://github.com/pydantic/pydantic/issues/568) issue, or define ignore for whole file instead of doing it in every validate function.
What do you think?

Reply:
Thanks for the information, I did not go deeper to find out the issue. Applied suggestion.